packages/office-ui-fabric-react/src/utilities/positioning.ts

```diff
@@ -581,10 +581,10 @@ export module positioningFunctions {
     const actualCallout = new Rectangle(0, callout.calloutRectangle.width, 0, callout.calloutRectangle.height);
 
     const calloutBounds = new Rectangle(
-      actualBeakWidth / 2,
-      callout.calloutRectangle.width - actualBeakWidth / 2,
-      actualBeakWidth / 2,
-      callout.calloutRectangle.height - actualBeakWidth / 2
+      beakWidth / 2,
```

Comment:
It would be good if you documented this code with the explanation you gave in this PR, to make it more understandable to other developers.

Reply:
I updated with more comments